app/controllers/classifier.coffee

```diff
@@ -90,6 +93,8 @@ class Classifier extends Controller
     @subjectViewer.setClassification @classification
     @animalSelector.setClassification @classification
 
+    AnalyticsLogger.logEvent User.current?.zooniverse_id,subject.zooniverseId,'view',''
```

Comment:
This firing only indicates that a subject was selected, not that the user actually viewed it. Not sure if that's what you're going for.